glossary.html

```diff
@@ -1,49 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-    
-   
-    </head>
-    
-    <body>
-    <header><nav>
-    
-    <ul>
-     <li><a href="index.html">Home</a></li>
-     <li><a href=" links.html">Link</a></li>
-     <li><a href="glossary.html">Glossary</a></li> 
-    </ul>
-   </nav>
-   </header>
+<head>
+
+    <title>YAOTIAN TONG's CSD120 Website</title> 
+
+</head>
+    
+<body>
+    <header>
+        <nav>    
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="links.html">Link</a></li>
+                <li><a href="glossary.html">Glossary</a></li> 
+            </ul>
+        </nav>
+    </header>
    
     <main>
     
-    <h1>Acronym Glossary</h1>
-    
-    	HTML
-    <dd>Hypertext Markup Language</dd>
-    	HTTP
-    <dd>Hypertext Transfer Protocol</dd>
-    	IP
-    <dd>Internet Proctocol</dd>
-    	TCP
-    <dd>Transmission Control Protocol</dd>
-    	URL
-    <dd>Uniform Resource Locator</dd>
+        <h1>Acronym Glossary</h1>
+        
+        <dl>
+        	<dt>HTML</dt>
+                <dd>Hypertext Markup Language</dd>
+        	<dt>HTTP</dt>
+                <dd>Hypertext Transfer Protocol</dd>
+        	<dt>IP</dt>
+                <dd>Internet Proctocol</dd>
+        	<dt>TCP</dt>
+                <dd>Transmission Control Protocol</dd>
+        	<dt>URL</dt>
+                <dd>Uniform Resource Locator</dd>
+        </dl>
     </main>
     
+    
     <footer>
-    <hr>
-    <blockquota>
-    		<dd>"The value of life lies not in the length of days, but in the use we make of them."<br></dd>
-    		<dd>- Montaigne</dd>
-    		<p>&copy;2019 <a  href="mailto:lilin5233@gmail.com">YAOTIAN TONG</a></p>
-    </blockquota>
+        <hr>
+        <blockquote>
+                "The value of life lies not in the length of days, but in the use we make of them."<br>
+                <em>&mdash; Montaigne</em>
```

Comment:
There should be no `<dd>` element here.  `<dd>` is only for definition lists.
The author should be emphasized using `<em>` rather than `<dd>`.
The — should use the `&mdash;` entity character